pkgs/by-name/de/degit-rs/package.nix

```diff
@@ -0,0 +1,41 @@
+{
+  lib,
+  fetchFromGitHub,
+  rustPlatform,
+  pkg-config,
+  openssl,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "degit-rs";
+  version = "0.1.3-unstable";
+
+  src = fetchFromGitHub {
+    owner = "psnszsn";
+    repo = pname;
+    rev = "c7dbeb75131510a79400838e081b90665c654c80";
+    hash = "sha256-swyfKnYQ+I4elnDnJ0yPDUryiFXEVnrGt9xHWiEe6wo=";
+  };
+
+  cargoPatches = [ ./add-Cargo.lock.patch ];
```

Comment:
I think there's no need to create a patch, the `cargo.lock` file can be linked in this derivation (if I am not mistaken).